src/components/molecules/FileUploader.jsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from 'react';
+import { Button } from '../atoms/Button';
+import "./FileUploader.css";
+
+const FileUploader = () => {
+  const [fileList, setFileList] = useState([]);
+  let inputRef;
```

Comment:
```suggestion
const inputRef = useRef()
```
ref를 사용할때는 useRef를 사용하세요